include/mull/MutationResult.h

```diff
@@ -22,6 +22,9 @@ class MutationResult {
   MutationPoint *getMutationPoint() { return MutPoint; }
   int getMutationDistance() { return distance; }
   Test *getTest() { return test; }
+  bool hasMutantSurvived() const {
+    return Result.status == ExecutionStatus::Passed;
+  }
```

Comment:
I'd say the `survived` is not the property that should exists on the level of `libmull`. There are at least two different ways to decide whether the mutant is survived, or not. I'd put this into a reporter.